packages/evm-gateway/src/lib.rs

```diff
@@ -1,10 +1,78 @@
-use ethers::prelude::abigen;
+use std::str::FromStr;
+
+use cosmwasm_std::{HexBinary, Uint128, Uint256};
+use ethers::{
+    abi::{encode, Token::Tuple, Tokenize},
+    prelude::abigen,
+    types::Address,
+};
+use sha3::{Digest, Keccak256};
+use thiserror::Error;
+
+use axelar_wasm_std::{hash::Hash, operators::Operators};
 
 abigen!(
     IAxelarAmplifierGateway,
     "src/abi/$SOLIDITY_GATEWAY_VERSION/IAxelarAmplifierGateway.json"
 );
```

Comment:
This generates the `WeightedSigners` type? Might be good to add a comment saying what types it generates